ff-netF48-server-sdk.csproj

```diff
@@ -8,10 +8,10 @@
         <PackageId>ff-dotnet-server-sdk</PackageId>
         <RootNamespace>io.harness.cfsdk</RootNamespace>
         <GeneratePackageOnBuild>false</GeneratePackageOnBuild>
-        <Version>1.7.0</Version>
```

Comment:
We haven't released 1.7.0 yet (aside from a release candidate) so we can safely release a patch for 1.6